24-nsm

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+. test.common
+
+test_start "nsm monitoring master node and slave node"
+nsm_node=3
+master_conf="hybrid_e2e		1
```

Comment:
fix spaces between attribute and value, either single space or alligned with other